### PR TITLE
Github Actions: change reviewers to team-reviewers and add CODEOWNERS

### PR DESCRIPTION
**Pull request by pmalek-sumo**
For some reason this doesn't change (not sure if the reason was new line separation - [docs mentions it should work](https://github.com/peter-evans/create-pull-request/#action-inputs))

### CODE REVIEW

**Comment by perk-sumo** [comment]
Maybe let's switch to `team-reviewers` and use the newly created https://github.com/orgs/SumoLogic/teams/opensource-collection-team ?

File: .github/workflows/automatic-prs-from-upstream.yml [hunk 1]
```diff
@@ -32,7 +32,4 @@ jobs:
 
             [1]: https://github.com/peter-evans/create-pull-request
           branch: upstream-changes
-          reviewers: |
-            pmalek-sumo
-            perk-sumo
-            sumo-drosiek
+          reviewers: pmalek-sumo,perk-sumo,sumo-drosiek
```
Comment (perk-sumo) [inline .github/workflows/automatic-prs-from-upstream.yml]:
```suggestion
          team-reviewers: @SumoLogic/opensource-collection-team
```

**Review by perk-sumo — APPROVED** [review]
I'd move to `team-reviewers` with https://github.com/orgs/SumoLogic/teams/opensource-collection-team but I'm good with this change as well.

File: .github/workflows/automatic-prs-from-upstream.yml (hunk 1 above)
Comment (pmalek-sumo) [reply .github/workflows/automatic-prs-from-upstream.yml]: Done

**Review by sumo-drosiek — APPROVED** [review]
Lets check it

**Comment by sumo-drosiek** [comment]
Please squash commits

**Pull request merged**